lib/core/pipes/app-config.pipe.ts

```diff
@@ -0,0 +1,31 @@
+/*!
+ * @license
+ * Copyright 2016 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Pipe, PipeTransform } from '@angular/core';
+import { AppConfigService } from './../app-config/app-config.service';
+
+@Pipe({
+    name: 'appConfig',
+    pure: true
```

Comment:
pipe are always pure by default https://angular.io/guide/pipes#pure-and-impure-pipes

Reply:
where do we use this new pipe?

Reply:
do we need it pure or impure?

Reply:
It is used in ACA right now, for example: 
```html
<adf-pagination
                [supportedPageSizes]="'pagination.supportedPageSizes' | appConfig"
                [target]="documentList"
                (changePageSize)="onChangePageSize($event)">
            </adf-pagination>
```

Reply:
I think `pure` (default) is okay. We usually need evaluating config values only once

Reply:
That is indeed the pipe Denis did for ACA. I initially planned to use it here, but I've ended not using it. I'll remove it, sorry.